xee/ext.py

```diff
@@ -1020,6 +1044,13 @@ def open_dataset(
         frameworks.
       ee_init_kwargs: keywords to pass to Earth Engine Initialize when
         attempting to auto init for remote workers.
+      use_coords_double_precision: Whether to use double precision for coordinates
```

Comment:
Should float64 just be the default? I think float32 was an offhand desire to save bytes, but it could be simply incorrect.

Reply:
Someone on the EE team will better be able to clarify.

Reply:
Yeah, I think this might be a good idea. If people want float32, they can cast before they write things out.